schema/deploy/warehouse/identifier/slices.sql

```diff
@@ -0,0 +1,13 @@
+-- Deploy seattleflu/schema:warehouse/identifier/slices to pg
+
+begin;
+
+alter table warehouse.identifier
+add column if not exists slices text[];
```

Comment:
I would remove the `if not exists` here (and the similar `if exists` in the revert). My tationale is that if the column exists when this change is applied, then something has gone wrong, and we'd want to know about it as an error condition, not as a (silent-ish) success.

Reply:
Agreed. I've changed that in the fixup commit.

Reply:
This is moot now that we're not storing slices in its own column. The "move to index" fixup commit has the new implementation.